.github/workflows/llama.yml

```diff
@@ -46,10 +46,13 @@ jobs:
 
     - name: Example
       run: |
-        cd wasmedge-ggml-llama
-        # curl -LO https://huggingface.co/kirp/TinyLlama-1.1B-Chat-v0.2-gguf/resolve/main/ggml-model-q4_0.gguf
+        cd wasmedge-ggml-llama-interactive
         curl -LO https://huggingface.co/juanjgit/orca_mini_3B-GGUF/resolve/main/orca-mini-3b.q4_0.gguf
         cargo build --target wasm32-wasi --release
-        wasmedge compile target/wasm32-wasi/release/wasmedge-ggml-llama.wasm wasmedge-ggml-llama.wasm
-        # wasmedge --dir .:. --nn-preload default:GGML:CPU:ggml-model-q4_0.gguf wasmedge-ggml-llama.wasm default '<|im_start|>user\nWhere is the capital of Japan?<|im_end|>\n<|im_start|>assistant\n'
-        wasmedge --dir .:. --nn-preload default:GGML:CPU:orca-mini-3b.q4_0.gguf wasmedge-ggml-llama.wasm default '### System:\nYou are an AI assistant\n\n### User:\nWhere is the capital of Japan?\n\n### Response:\n'
+        wasmedge compile target/wasm32-wasi/release/wasmedge-ggml-llama-interactive.wasm wasmedge-ggml-llama-interactive.wasm
```

Comment:
We can remove this line. The bottleneck of GGML is the CPU and model. AOT does rare help here.

Reply:
Done.